corrector.rb

```diff
@@ -0,0 +1,10 @@
+class Corrector
+  def correct_name(name)
+    given_name = name.strip.capitalize
+    if given_name.length > 10
+      given_name[0, 10]
+    else
+      given_name
+    end
```

Comment:
[OPTIONAL]

- You could directly pass `given_name[0, 10]` and it will directly take the name and If it's longer it trim the word to 10 characters. :smile:

Reply:
Thank you for letting me know. I made the changes.